CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Added:
  * Add unstable `otel_metrics_ingestion` resource.
+ * Add `group_by` to `notification_routes` resource.
```

Comment:
nit: consider "Add optional `group_by` to notification routes in `notification_policy` resource" or similar.